lib/stripe/account.rb

```diff
@@ -22,7 +23,7 @@ def self.retrieve(id=nil, opts={})
         opts = id
         id = nil
       end
-      super(id, opts)
+      super(id, Util.normalize_opts(opts))
```

Comment:
This looks like another unnecessary call to `normalize_opts`, since `super` already calls it.

(After this, I think the PR will be ready to merge -- sorry for the delay.)